apps/assisted_tagging/dataloaders.py

```diff
@@ -0,0 +1,23 @@
+from collections import defaultdict
+
+from promise import Promise
+from django.utils.functional import cached_property
+from assisted_tagging.models import AssistedTaggingPrediction
+
+from utils.graphene.dataloaders import DataLoaderWithContext, WithContextMixin
+
+
+class DraftEntryPredicationsLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        assisted_tagging_qs = AssistedTaggingPrediction.objects\
+            .filter(draft_entry_id__in=keys, is_selected=True).only('draft_entry_id')
+        _map = defaultdict(list)
+        for assisted_tagging in assisted_tagging_qs:
+            _map[assisted_tagging.draft_entry.id].append(assisted_tagging)
```

Comment:
Let's not introduce N+1 when trying to fix it
```suggestion
            _map[assisted_tagging.draft_entry_id].append(assisted_tagging)
```